Makefile

```
@@ -25,6 +25,10 @@ cortex-m: cortex-m.Dockerfile
 	cat $< | sed "s/modm-build-base:TAG/modm-build-base:$(DATE)/g" | \
 	docker build --tag ghcr.io/modm-ext/modm-build-$@:$(DATE) -
 	docker tag ghcr.io/modm-ext/modm-build-$@:$(DATE) ghcr.io/modm-ext/modm-build-$@:testing
+cortex-m-openocd: cortex-m-openocd.Dockerfile
```

Your new image depends on the `cortex-m` image:

```suggestion
cortex-m-openocd: cortex-m-openocd.Dockerfile cortex-m
```